### ISSUE TITLE
Initial creation of mobile scoreboard

### ISSUE THREAD

**Issue by jawaad-ahmad** [brata-hsdc/brata.masterserver#8]
Enhance the legacy MS as follows:

Make a mobile version of the leader board, and maybe an individual team status board, that a team can access from their mobile device.

An idea we had was to make the **phone leaderboard** just have the team status. This will give all the teams the status they would need without requesting an admin to get it manually from the master server (if an app crashes, they can check # of retries to know if data was sent to the server).

(I think anything front-end should be reviewed with mock-ups before we actually start coding it. If anything changes on paper, it's a lot less effort than reworking code.)

This breaks up Issue brata-hsdc/brata#74 Part 5. Issue brata-hsdc/brata#74 was later continued as Issue #4.

**Comment by ellerychan** [comment]
See my comment about CSS in Issue #7.

**Comment by jawaad-ahmad** [comment]
I would recommend this to go down to `low` priority. It's not necessary till we get all the other stuff working, but it's still good to keep in-mind up-front to make sure we don't box ourselves into a desktop scoreboard only.

**Comment by ellerychan** [comment]
I was wondering whether all the team <--> ms interactions should be handled by **teamcentral** or whether the mandatory interactions (team registration, messages that are sent as part of a competition event, etc.) are handled by another part, and **teamcentral** is just for optional team status.  I can't think of another part that is more appropriate.

I agree that the info/status/entertainment interface could be given a lower priority.

**Comment by jawaad-ahmad** [comment]
Good point. We would need to retain the current functionality of Mike registering the teams manually in case someone's phone is having trouble, but if we push as much self-registration work as possible to the phones, that frees up the Mike and reduces the registration bottleneck.

What interactions do we need to support, and what do we want to support for additional automation? (We might want to open dedicated issues for each of these.)
- Self-registration vs. manual registration
- Mobile scoreboard (Issue #8)
- Maybe some sort of self-service bypass request (ties into Issue #9)
  1. Escort gives the OK to go into this area.
  2. Students enter and submit information.
  3. Meanwhile escort calls Jaron and explains the situation.
  4. Jaron gives the OK and presses a button somewhere, or looks up some code for students to enter.
  5. Students enter the code to bypass the challenge knowing this action will reduce their score or give them zero credit.
  6. Students proceed to the next challenge.
- Trivia game or some other entertainment while waiting

**Comment by jawaad-ahmad** [comment]
@ellerychan I assume this one is still low-priority; we're not working on this at the moment. Correct?

**Comment by ellerychan** [comment]
Correct.
